### PR TITLE
Add sops

### PR DESCRIPTION
**Pull request by lgarciaaco**

### CODE REVIEW

File: observability/sops/alerts/strimzi_operator_reconciliations_failed.asciidoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+== Troubleshooting
+
+* Check the logs of the connector and follow up on any errors you notice.
+* Check the logs of the Strimzi operator looking for any specific entry related to the connector ID.
+* Check if the Kafka cluster expired and doesn't exist anymore.
+
+// Include the following step in every alert SOP
+* If the above hasn't worked contact engineering:
+** Use the MK - Supporting Service Gchat channel during business hours.
```
Comment (heiko-braun) [inline observability/sops/alerts/strimzi_operator_reconciliations_failed.asciidoc:65]: Do we want to advertise the `MK Supporting` chat channel here?

File: observability/sops/alerts/strimzi_operator_reconciliations_failed.asciidoc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+== Description
+
+The purpose of this SOP is to describe the process of resolving the alert `StrimziOperatorReconciliationsFailed`.
+
+// Include the following step in every alert SOP, changing required parts
+The source of the alert is the `strimzi_reconciliations_failed_total` metric. It fires when the increase in the time series in the range vector is greater than 5 in the last hour: `sum(increase(strimzi_reconciliations_failed_total[1h])) > 5`
+
+When this alert triggers it means that in the past hour, strimzi connectors have failed to reconcile more than 5 times.
+
+A reason for connectors to fail reconciling is, that the Kafka cluster is unreachable.
```
Comment (rk3rn3r) [inline observability/sops/alerts/strimzi_operator_reconciliations_failed.asciidoc:26]: `RHOSAK cluster`or `RHOSAK Kafka custer`?

File: observability/sops/alerts/strimzi_operator_reconciliations_failed.asciidoc [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+== Execute/Resolution
+
+// Include this as the first step in every alert SOP
+. Log in to the OSD cluster that generated the alert. You can identify the cluster by looking at the field `Source` in the alert.
+
+. The *redhat-openshift-connectors-observability / Strimzi Operators* Grafana dashboard can help you to troubleshoot the issue.
+
+. Identify the reason why the Operator can't reconcile the connector CR.
+
+.. If the cause is that the Kafka cluster isn't available because it expired, proceed to https://github.com/bf2fc6cc711aee1a0c2a/cos-tools/blob/main/observability/sops/howto/stop_connector.asciidoc[stop the connector].
```
Comment (rk3rn3r) [inline observability/sops/alerts/strimzi_operator_reconciliations_failed.asciidoc:44]: We should eventually add how to determine that.

**Comment by rk3rn3r** [comment]
LGTM

**Pull request merged**